XROLE/storeManus
# CH(create test for unit validation functions)

####what doess this pull request do?
- Test validation functions

#### Description of task completed? 
-- Test isNumber() method to see that it returns true when a number is imputed and vice vasa
- Test isValidNum() method to see that it returns true only when number.lenght equals 13
- Test isEmpty() method to see that it returns true only when empty string is entered as value.

#### How should this be tested?
- clone the repo
- run npm install to install the dependencies
- run npm test to see informative message telling that the app passed

#### Any background context you will want to provide?
N/A

#### what are the relevant pivotal tracker stories?
- Story type - Chore
- Story title - Test validation functions
- Story ID - #161502756
